config/logger.go

```diff
@@ -36,12 +37,17 @@ type bugsnagHook struct{}
 func (b bugsnagHook) Levels() []logrus.Level {
 	return []logrus.Level{
 		logrus.ErrorLevel,
+		logrus.FatalLevel,
 		logrus.PanicLevel,
-		logrus.WarnLevel,
 	}
 }
 
 // Fire sends the logrus entry to bugsnag.
 func (b bugsnagHook) Fire(entry *logrus.Entry) error {
-	return bugsnag.Notify(errors.New(entry.Message), entry.Data)
+	err, ok := entry.Data[logrus.ErrorKey].(error)
+	if !ok {
+		err = errors.New(entry.Message)
+	}
+	notify := bugsnagErrors.New(err, 1)
```

Comment:
What's 1?

Reply:
It's how much stacktrace to skip (this method). In theory we could cut off logrus from the stacktrace - but it's a big unknown since we don't control the code. I think it'll be better to provide our own stacktraces in future
https://godoc.org/github.com/bugsnag/bugsnag-go/errors#New

Reply:
Chuck a comment about that in there then LGTM 👍

Reply:
Done